AdditionalTask2/AdditionalTask2/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Globalization;
+
+namespace AdditionalTask2;
+
+public class PositivOrNegative
+{
+    public static void Main()
+    {
+        IFormatProvider formatter = new NumberFormatInfo { NumberDecimalSeparator = "." };
+        Console.WriteLine("Hello! I'll try to recognize if the number is positive, negative or 0."
+            + "\nUse symbol \".\" to make your number float!");
+        Console.WriteLine("Please, enter your number:");
+        string? userInput = Console.ReadLine();
```

Comment:
Why you have decided to use nullable string?

Reply:
because there was warning ))